### PR TITLE
add delete-sessions to controller

### PR DESCRIPTION
**Pull request by andrewpmartinez**
Example Output: 
```
[   0.018]    INFO ziti/ziti-controller/subcmd.deleteSessions: {go-version=[go1.16] build-date=[2020-01-01 01:01:01] revision=[local] os=[windows] arch=[amd64] version=[v0.0.0]} removing API Sessions and Edge Sessions from ziti-controller
[   0.023]    INFO ziti/ziti-controller/subcmd.deleteSessions.func2: Existing API Sessions: 3
[   0.023]    INFO ziti/ziti-controller/subcmd.deleteSessions.func2: Existing Edge Sessions: 5
[   0.023]    INFO ziti/ziti-controller/subcmd.deleteSessions.func3: Done removing API Sessions
[   0.023]    INFO ziti/ziti-controller/subcmd.deleteSessions.func3: Done removing Edge Sessions
```

Example Output (if ran again): 
```
[   0.018]    INFO ziti/ziti-controller/subcmd.deleteSessions: {go-version=[go1.16] os=[windows] arch=[amd64] build-date=[2020-01-01 01:01:01] revision=[local] version=[v0.0.0]} removing API Sessions and Edge Sessions from ziti-controller
[   0.023]    INFO ziti/ziti-controller/subcmd.deleteSessions.func2: API Session bucket does not exist, skipping, count is: 0
[   0.023]    INFO ziti/ziti-controller/subcmd.deleteSessions.func2: Edge Sessions bucket does not exist, skipping, count is: 0
```

Help:
```
> ziti-controller -h
Ziti Fabric Controller

Usage:
  ziti-controller [command]

Available Commands:
  delete-sessions Delete all API Sessions and Edge Sessions, controller must be shutdown
  edge            used to perform various edge related functionality
  help            Help about any command
  run             Run controller configuration
  version         Show component version

Flags:
      --cli-agent-addr string   Specify where CLI Agent should list (ex: unix:/tmp/myfile.sock or tcp:127.0.0.1:10001)
  -a, --cliagent                Enable/disabled CLI Agent (enabled by default) (default true)
  -h, --help                    help for ziti-controller
      --log-formatter string    Specify log formatter [json|pfxlog|text]
  -v, --verbose                 Enable verbose logging

Use "ziti-controller [command] --help" for more information about a command.
```

### CODE REVIEW

**Comment by andrewpmartinez** [comment]
Need to add release notes

**Comment by plorenz** [comment]
Should probably clean the token indexes out as well. Otherwise looks good

File: ziti-controller/subcmd/delete_sessions.go [hunk 1]
```diff
@@ -0,0 +1,137 @@
...
+			root := tx.Bucket([]byte("ziti"))
+
+			if root == nil {
+				return errors.New("root 'ziti' bucket not found")
+			}
+
+			apiSessionBucket := root.Bucket([]byte(ApiSessionBucketName))
+
+			if apiSessionBucket == nil {
+				logger.Info("API Session bucket does not exist, skipping, count is: 0")
```
Comment (michaelquigley) [inline ziti-controller/subcmd/delete_sessions.go:75]: Stylistically, golang in general, and ziti usually, have typically used all lowercase in log messages.

Comment (andrewpmartinez) [reply ziti-controller/subcmd/delete_sessions.go]: Moved them all to lower case

**Review by plorenz — APPROVED** [review]

**Pull request merged**